src/todo/CreateTodo.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function InputAndBtn({setInitialText, clicked}){
```

Comment:
Callbacks, aka function props, are usually named `onSomething`.
Also, you don't need function props for setting initial values, that's why you have regular props.